README.md

```diff
@@ -142,3 +142,24 @@ To prevent this you can add this code to specify the PHP version you want to use
     "platform": {"php": "5.5.9"}
 },
 ```
+### How can I add libraries using composer.json?
+
+It is possible to use libraries with composer thanks to the
+asset-packagist repository (https://asset-packagist.org/).
+
+For example, to use colorbox:
+```
+composer require npm-asset/colorbox:"^0.4"
```

Comment:
I'd also mention chosen module and its JS dependency as an example, because if you install the library from npm-asset repo then you have to add a library specific override to the installer-paths, otherwise library gets installed with an incorrect name `chosen-js`.
Required line: ` "docroot/libraries/chosen": ["npm-asset/chosen-js"],`
See:
* https://cgit.drupalcode.org/chosen/tree/README.txt?h=8.x-2.x#n77
* https://cgit.drupalcode.org/chosen/tree/README.txt?h=8.x-2.x#n77